config/logit/30_various_timestamps.conf

```diff
@@ -0,0 +1,116 @@
+date {
```

Comment:
Add comment for context

```suggestion
# Use more accurate timestamp fields, instead of the timestamp of when logit received the log

date {
```